packages/block-editor/src/components/navigable-toolbar/index.js

```diff
@@ -35,11 +35,27 @@ function focusFirstTabbableIn( container ) {
 }
 
 function useIsAccessibleToolbar( ref ) {
+	/*
+	 * By default, we'll assume the starting accessible state of the Toolbar
+	 * is true, as it seems to be the most common case.
+	 *
+	 * Transitioning from an (initial) false to true state causes the
+	 * <Toolbar /> component to mount twice, which is causing undesired
```

Comment:
Mount or render?

Reply:
mount :)

Reply:
I wonder if this is still needed after the adjustments I made?